refined/shared/src/main/scala/monocle/refined/strings.scala

```diff
@@ -0,0 +1,17 @@
+package monocle.refined
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.string.StartsWith
+import monocle._
+
+object strings extends StringsInstances
+
+trait StringsInstances {
+
+  def startsWith(prefix: String): Prism[String, StartsWithString[prefix.type]] = {
+    Prism.partial[String, Refined[String, StartsWith[prefix.type]]] {
+      case string if string.startsWith(prefix) => Refined.unsafeApply(string)
```

Comment:
refined already knows how to check if a `String` satisfies the `StartsWith[prefix.type]` predicate via its `Validate` type class. If the `startsWith` method would take an `implicit v: Validate[String, StartsWith[prefix.type]]` the check `string.startsWith(prefix)` could be replaced with `v.isValid(prefix)`. This would be my first step to generalize the refined `Prism` definitions.